src/EasyList/ParseAdd.cs

```diff
@@ -54,7 +54,7 @@ public static class ParseAdd
             string label = GetData("add", args, positions) ?? string.Empty;
             string description = GetData("-d", args, positions) ?? string.Empty;
             string dueDate = GetData("-t", args, positions) ?? string.Empty;
-            string priority = GetData("-p", args, positions) ?? "${TodoPriority.Low}";
```

Comment:
How did this get committed? Was the code not debugged?

Reply:
Yup, Maybe Expecting only a silly string change I didn't check after this commit.